### PR TITLE
Implement mqtt client reference

### PR DESCRIPTION
**Pull request by morsdyce**

### CODE REVIEW

**Comment by borisd** [comment]
Task linked: [CU-2kb6yf7 MQTT](https://app.clickup.com/t/2kb6yf7)

File: src/todo.ts [hunk 1]
```diff
@@ -32,6 +33,21 @@ export const removeLicense = async (license: Licence) => {
   }
 };
 
+export const upgradeFirmware = async (firmwareUrl: string) => {
```
Comment (Chikel) [inline src/todo.ts:36]: Should we add this (and line 160 fix) to Vladimir's work as well?

File: package.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
   "dependencies": {
     "compare-versions": "4.1.3",
     "forever-monitor": "3.0.3",
-    "isomorphic-fetch": "3.0.0"
+    "isomorphic-fetch": "3.0.0",
+    "mqtt": "^4.3.7"
```
[the remark this answers is not in the document]
Comment (morsdyce) [reply package.json:29]: Yes, there won't be any breaking changes

File: src/todo.ts (hunk 1 above)
Comment (morsdyce) [reply src/todo.ts:36]: line 160 for sure but I already told him about it. This i'm not sure it shows dealing with firmware upgrades who were not done via commands. not all the api is implemented in both of the repos

**Pull request closed without merging**